### DISCUSSION TITLE
MODIS Sea ice data (no credentials from forces2021)

### DISCUSSION THREAD

**Discussion by dominichr1** [pangeo-data/escience-2022#15] [Q&A]
Haven't got the credentials to access the MODIS data, as in the example script:

`fs = s3fs.S3FileSystem(anon=False,
client_kwargs={'endpoint_url': 'https://forces2021.uiogeo-apps.sigma2.no/'})`

https://forces2021.uiogeo-apps.sigma2.no/

Error  message: @keitsuruhara 

NoCredentialsError: Unable to locate credentials

**Comment by MichaelSchulzMETNO** [comment]
Ja, the sigma2 store does not work unfortunately. To my knowledge. We can download data to metno, if desired.

**Comment by sarambl** [comment]
There's a lot of sea ice data in the Ada's bucket!
